Worksheet_C/README.md

```diff
@@ -0,0 +1,67 @@
+***
+
+a) This algorithm looks through both list I and j and checks whether I is not equal to j and whether list[i] is equal to list [j]if they are, then the list has a duplicate within it and the code returns
```

Comment:
There is only one list -- i and j are not lists, they are indexes into the list.